packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.2
+
+* Updates minimum version to iOS 11.
```

Comment:
The [README](https://github.com/flutter/plugins/blob/main/packages/ios_platform_images/README.md?plain=1#L13) needs an update too. We haven't built out tooling to enforce that it matches reality yet (https://github.com/flutter/flutter/issues/84200).